src/OpenTelemetry/Metrics/HistogramMeasurements.cs

```diff
@@ -16,7 +16,7 @@
 
 namespace OpenTelemetry.Metrics
 {
-    internal class HistogramMeasurements
+    public class HistogramMeasurements
```

Comment:
```suggestion
    public class HistogramMetricPoint
```

Reply:
Is this a `HistogramMetricPoint` or it is a `HistogramMetricPointBucket`

Reply:
HistogramMetricPointBucket is correct.

Reply:
`HistogramBuckets` - this sounds better name.